BUILDING.md

```diff
@@ -573,6 +573,20 @@ export CC="ccache cc"          # add to ~/.zshrc or other shell config file
 export CXX="ccache c++"        # add to ~/.zshrc or other shell config file
 ```
 
+On Windows:
+
+Tips: follow <https://github.com/ccache/ccache/wiki/MS-Visual-Studio>, and you
+should notice that obj file will be bigger the normal one.
+
+First, install ccache, assume ccache install to c:\ccache, copy
```

Comment:
IIUC, thinks means that if your ccache is installed at `c:\path\to\ccache`, then you can just do

```
cp c:\path\to\ccache\ccache.exe c:\path\to\ccache\cl.exe
.\vcbuild.bat ccache c:\ccache\
```

?

(We recommend installing Git and the UNIX tools added to PATH on Windows, so it seems fine to assume `cp` is available).

Reply:
Ah yes, that makes sense. I'll add the command about the `cp` command to the doc.